integrationtest/src/test/resources/hotrod-client-testingconfiguration-offset100.properties

```diff
@@ -0,0 +1,25 @@
+# hopefully this port is available
+infinispan.client.hotrod.server_list = 127.0.0.1:11322
+
+infinispan.client.hotrod.marshaller = org.hibernate.ogm.datastore.infinispanremote.impl.protostream.OgmProtoStreamMarshaller
+infinispan.client.hotrod.async_executor_factory = org.infinispan.client.hotrod.impl.async.DefaultAsyncExecutorFactory
+infinispan.client.hotrod.default_executor_factory.pool_size = 1
+infinispan.client.hotrod.default_executor_factory.queue_size = 100
+infinispan.client.hotrod.tcp_no_delay = true
+infinispan.client.hotrod.tcp_keep_alive = false
+
+#TODO could we estimate these in advance from our knowledge of the data model?
+infinispan.client.hotrod.key_size_estimate = 64
+infinispan.client.hotrod.value_size_estimate = 512
+#This is critical: [FIXME check for this to be set?]
+infinispan.client.hotrod.force_return_values = true
```

Comment:
Is this necessary? I think we already validate the expected values in `InfinispanRemoteConfiguration`. Just asking because, if that's the case, the comment is misleading